solvcon/tests/test_domain.py

```diff
@@ -133,6 +133,45 @@ def test_splitted_neibcl(self):
             self.assertTrue((blk.fccls[slct,3] == -1).all())
 
     def test_splitted_bnds(self):
+        """
+        splitted number of boundries.
+
+        self.dom is <class 'solvcon.domain.Collective'>.
```

Comment:
Sphinx style is to write :py:class:`solvcon.domain.Collective`, but it's OK to use repr format here.